turbo/execution/eth1/block_building.go

```diff
@@ -200,10 +200,16 @@ func (e *EthereumExecutionModule) GetAssembledBlock(ctx context.Context, req *ex
 	}
 
 	var requestsBundle types2.RequestsBundle
-	if blockWithReceipts.Requests != nil && len(*blockWithReceipts.Requests) > 0 {
-		requests := make([][]byte, len(*blockWithReceipts.Requests))
-		for i, r := range *blockWithReceipts.Requests {
-			requests[i] = r.RequestData
+	if blockWithReceipts.Requests != nil {
```

Comment:
Is this change necessary?

Reply:
While it's true that I am making sure that the built block will have the empty requests alright, this is additional padding on it. The `else` logic can be gotten rid of though, later. The change makes this demarcation.